src/otk/annotation.py

```diff
@@ -176,29 +177,25 @@ def squash_annotations(cls, annotation_list: list) -> dict:
 
 
 class AnnotatedList(Generic[AnnotatedListT], AnnotatedBase, list):
-    def __init__(self, seq: list[Any] | None = None) -> None:
+    def __init__(self, seq: Optional[list[Any]] = None) -> None:
```

Comment:
This change (and the other type annotations in this commit) doesn't seem relevant to the commit and its message.